### PR TITLE
Fix the event for new version of Classic Era

### LINKED ISSUE #32: [1.14.4] Addon doesn't show bags and gives out Lua errors
Hi! I'm having multiple errors on Hardcore Era.

1. Multiple `SharedXML` errors:

	<details>
	<summary>Details</summary>
	
	```lua
	18x SharedXML/SharedUIPanelTemplates.xml:1474 Texture Texture: Unable to load atlas entry UI-Frame-DiamondMetal-Header-CornerRight
	```
	
	</details>

2. `CURSOR_UPDATE` error:

	<details>
	<summary>Details</summary>
	
	```lua
	1x InventorianBag2:RegisterEvent(): InventorianBag2:RegisterEvent(): Attempt to register unknown event "CURSOR_UPDATE"
	[string "=[C]"]: in function `RegisterEvent'
	[string "@Inventorian/Bag.lua"]:88: in function `Set'
	[string "@Inventorian/Frame.lua"]:214: in function `UpdateBags'
	[string "@Inventorian/Frame.lua"]:202: in function `ToggleBagFrame'
	[string "@Inventorian/Frame.lua"]:127: in function `OnBagToggleClick'
	[string "*Frame.xml:242_OnClick"]:1: in function <[string "*Frame.xml:242_OnClick"]:1>
	
	Locals:
	(*temporary) = InventorianBag2 {
	 0 = <userdata>
	 hasItem = "|cffffffff|Hitem:4496::::::::14:::::::::|h[Small Brown Pouch]|h|r"
	 count = InventorianBag2Count {
	 }
	}
	(*temporary) = "CURSOR_UPDATE"
	```
	
	</details>

@Nevcairiel any chance you could look into this?

### PR DESCRIPTION
**Pull request by nezorflame**
Fixes #32

### CODE REVIEW

File: Bag.lua [hunk 1]
```diff
@@ -82,11 +82,7 @@ function Bag:Set(parent, id)
 		self:Update()
 
 		self:RegisterEvent("ITEM_LOCK_CHANGED")
-		if select(4, GetBuildInfo()) >= 30400 then
-			self:RegisterEvent("CURSOR_CHANGED")
-		else
-			self:RegisterEvent("CURSOR_UPDATE")
-		end
+		self:RegisterEvent("CURSOR_CHANGED")
```
Comment (nezorflame) [inline Bag.lua:85]: removed the `if` since every version of WoW now has the same event

**Comment by naxmefy** [comment]
dont forget

image

**Comment by Splitix** [comment]
Confirmed this fixes the issue in Classic Era

**Comment by nezorflame** [comment]
Not sure why this wasn't merged, but it seems that @Nevcairiel has fixed the issue in the source with https://github.com/Nevcairiel/Inventorian/commit/64ee1ded8231bf378cc40fe3d5f656c0149f414e

**Pull request closed without merging**